pyproject.toml

```diff
@@ -22,37 +23,43 @@ readme="README.md"
 exclude = ["tests/*", "docs/*"]
 
 [tool.poetry.dependencies]
-dacite = "*"
-dask = "*"
-dask_cuda = "*"
-dask_jobqueue = "*"
-dask_memusage = "*"
-dask_ml = "*"
-dask-pytorch-ddp = "*"
-GPUtil = "*"
+bokeh = "2.4.3"
+charset-normalizer = "<3.0"
+dacite = "1.8.1"
+dask = "2023.3.2"
+dask_cuda = "23.4.0"
+dask_jobqueue = "0.8.2"
+dask_memusage = "1.1"
+dask_ml = "2023.3.24"
+dask-pytorch-ddp = "0.2.2"
+GPUtil = "1.4.0"
 gdown = "^4.6"
-graphviz = "*"
-h5py = "*"
-hdbscan = "*"
-ipympl = "*"
-matplotlib = "*"
-memray = "*"
-networkx = "*"
-ormsgpack = "*"
-packaging = "*"
-portalocker = "*"
-protobuf = "~=3.20.1"
-psutil = "*"
-pyarrow = "*"
+graphviz = "0.20.1"
+h5py = "3.10.0"
+hdbscan = "0.8.33"
+ipympl = "0.9.3"
+joblib = "1.1.1"
+matplotlib = "3.7.1"
+memray = "1.10.0"
+networkx = "3.2.1"
+numpy = "1.23.5"
+ormsgpack = "1.4.1"
+packaging = "23.1"
+pandas = "1.5.2"
+portalocker = "2.8.2"
+protobuf = "<=3.20.1"
+psutil = "5.9.4"
+pyarrow = "10.0.1.dev0+ga6eabc2b.d20230609"
 python = "^3.8"
 pytorch-lightning = "*"
-scikit-learn = "*"
-torchvision = "*"
-xarray = "*"
-xgboost = "*"
-zarr = "*"
+scikit-learn = "1.2.0"
+torchvision = "0.16.0a0"
+tornado = "<6.2"
+xarray = "2023.11.0"
+xgboost = "1.7.5"
+xpysom-dask = { git = "https://github.com/jcfaracco/xpysom-dask.git", tag = "v1.0.7.1" }
```

Comment:
@jcfaracco is this the latest version of this library?

Reply:
Yes, this is the last one :-) but the branch is `dask`.